src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final int RED = 1;
+    private static final int YELLOW = 2;
+    private static final int GREEN = 3;
```

Comment:
If we create COLORS array we don't need this any more

Reply:
I deleted that.